pkg/catalog/outbound_traffic_policies.go

```diff
@@ -351,3 +371,38 @@ func (mc *MeshCatalog) ListMeshServicesForIdentity(identity identity.ServiceIden
 
 	return dstServices
 }
+
+func getRetryPolicy(retryPolicy trafficpolicy.RetryPolicy, key, val string) (bool, int) {
```

Comment:
The function is named `getRetryPolicy` but is returning `(bool, int)`. So I suggest refactoring the usage of this function.

The caller of this function should be changed to do something similar to:
```
retryPolicy, err = getRetryPolicyForTrafficTarget(t)
...
```